src/stores/products.ts

```diff
@@ -0,0 +1,75 @@
+import { getProduct } from "@api/getProduct";
+import { getProducts } from "@api/getProducts";
+import { Product } from "@api/types";
+import { normalizeList } from "@utils/normalizeList";
+import { makeObservable, observable, action } from "mobx";
+
+const LIMIT = 9;
+
+type Ids = number[];
+type Entities = Record<number, Product | undefined>;
```

Comment:
а почему тут `| undefined`? Вроде достаточно
```suggestion
type Entities = Record<number, Product>;
```